### PR TITLE
Added Style

### LINKED ISSUE #5: Add Style
#### As a user I want to see a visually pleasing application.

__Story__:

Our web application is really starting taking shape. We have added a few
features that provide value to our customers, but some of our customers are
complaining that the website is a quite bland. It is time to spice it up by
applying a visually appealing, and easy to maintain user interface to our web
application.

__Acceptance criteria__:
- [x] The application has a header with navigation
- [x] Images are displayed in an aesthetically pleasing list of cards
- [x] Form errors are shown inline with the related input field
- [x] I know how to use Bootstrap components like buttons and navbars in my
  application.
- [x] I know the very basics of CSS and SCSS
- [x] My SCSS files are properly named and organized using BEM convention
- [x] My app has a "sticky" footer.
- [x] My app has a few customized styles using bootstrap variable overrides.
- [x] My app is responsive and is usable on a phone.
- [x] The active link in the navigation header is differentiated.
- [x] I use a bootstrap "container" to center my app contents (not necessarily "fluid").

__Reference material__:
- http://v4-alpha.getbootstrap.com/getting-started/introduction/
- https://github.com/plataformatec/simple_form
- http://getbem.com/naming/

__Dependencies__:
- Image Index

### PR DESCRIPTION
**Pull request by heinzsamuel**
Closes #5

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -18,15 +18,19 @@
   <nav class="navbar navbar-expand-lg navbar-light bg-light" >
     <div class="container-fluid">
       <div class="navbar-header">
-        <a class="navbar-brand" href="#">Image Sharer</a>
+        <div class="navbar-brand" href="#">Image Sharer</div>
```
Comment (almgong) [inline app/views/layouts/application.html.erb]: Should we make this `div` contain a link?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -18,15 +18,19 @@
...
       </div>
       <div class="nav navbar-nav">
-        <div class="nav-item nav-link active">
-          <%= link_to 'Index', images_path %>
-        </div>
-        <div class="nav-item nav-link">
-          <%= link_to 'New', new_image_path %>
-        </div>
+        <% if current_page?(images_path) %>
```
Comment (almgong) [inline app/views/layouts/application.html.erb]: This logic is sound, but I wonder if we could reduce the amount of repeated code to achieve what we want. Perhaps we could check `current_page?(...)` closer to what differs across conditions?

File: app/views/images/index.html.erb [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+  <h1>Images#index</h1>
+  <div class='container'>
   <% @images.each do |image| %>
-    <tr>
-      <h1 class="js-image-title"><%= image.title %></h1>
-      <%= image_tag(image.url, :alt => 'linked image', width: '400px') %>
-
-    </tr>
+    <div class='card'>
+      <div class='card-body'><p class="js-image-title card-text"><%= image.title %></p></div>
```
Comment (almgong) [inline app/views/images/index.html.erb]: Do the classes `class-body` and `card-text` have style rules defined? (as well as `card-img-top` defined below)

Comment (almgong) [reply app/views/images/index.html.erb]: Disregard this, realized that these are Bootstrap classes defined for cards 😄

Comment (heinzsamuel) [reply app/views/images/index.html.erb]: All the classes card-* have defined style rules in the default bootstrap stylesheet

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (heinzsamuel) [reply app/views/layouts/application.html.erb]: Good point. It is now pointed to the index page.

**Pull request merged**